contents/blog/gdpr-google-analytics-privacy-shield.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2022-01-25
+title: A new 'Privacy Shield' won't solve big tech's GDPR problem
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/google-analytics-gdpr.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Privacy", "Product Analytics", "General"]
+---
+
+Ten years ago today, the European Commission published the first draft of the General Data Protection Regulation (GDPR). But, as [the recent ruling in Austria concerning one website's use of Google Analytics](https://isgoogleanalyticsillegal.com/) proves, how those rules are enforced and interpreted is still contested.
+
+There's lots of detail in the [full ruling](https://noyb.eu/sites/default/files/2022-01/E-DSB%20-%20Google%20Analytics_EN_bk.pdf), but here's a quick summary:
+
+- An Austrian website was found to have exported personal data of a user to Google servers in the US
+- The complaint argued this was a breach as it was subject to demands from local intelligence agencies
+- The Austrian Data Protection Agency agreed it was a breach and the case remains ongoing 
+
+That's a very simplified version of events, but here are three important takeaways:
+
+### 1 . Google itself was cleared of any wrongdoing
+Only the website that was using Google Analytics, as the data controller, was found liable for the breach. Under the GDPR, breaching the regulation can result in huge fines of up to €20 million or 4% of a company's global turnover, whichever is the largest. [Amazon was fined €746 million by authorities in Luxembourg in 2020](https://www.bbc.co.uk/news/business-58024116), the largest GDPR fine so far. Getting it wrong can be painful.
+
+### 2. Any website using GA in the EU could be liable
```

Comment:
Does this mean US websites with EU visitors? 

What do you specifically mean by "using Google Analytics in the EU"?

Reply:
Strictly speaking, it could mean any website using GA who collects data on EU citizens, so perhaps 'Any website using GA could be liable' is more accurate?

Reply:
could we just rephrase to that then? Ie "any website using GA, collecting data on EU citizens"

Reply:
(generally, being a bit more specific as long as we're correct is a good thing - our audience love the actual details where we can provide them without legal risk!)